components/speedreader/BUILD.gn

```diff
@@ -40,7 +32,10 @@ source_set("speedreader") {
     "//brave/components/brave_component_updater/browser",
     "//brave/components/resources",
     "//brave/components/resources:static_resources_grit",
+    "//brave/components/speedreader/common",
     "//brave/components/weekly_storage",
+    "//components/content_settings/core/browser",
```

Comment:
would be great to also fix DEPS

Reply:
done. tagged DEPS_REVIEWERS though

Reply:
I think components/speedreader/DEPS also needs new entries

Reply:
fixed